modules/ossm-installing-multi-primary-network-mesh.adoc

```diff
@@ -0,0 +1,290 @@
+:_mod-docs-content-type: Procedure
+[id="ossm-installing-multi-primary-multi-network-mesh_{context}"]
+= Installing a multi-primary multi-network mesh 
+:context: ossm-installing-multi-primary-multi-network-mesh
+
+Install {istio} in the multi-primary multi-network configuration on two {ocp-product-title} clusters. 
```

Comment:
In the other doc, we call this the "multi-primary multi-network topology", so it might make sense to update this.